Car_owners/car_owners/wsgi.py

```diff
@@ -1,6 +1,14 @@
+import dotenv
 import os
 from django.core.wsgi import get_wsgi_application
+from pathlib import Path
 
+CURRENT_DIR = Path(__file__).resolve().parent
+BASE_DIR = CURRENT_DIR.parent
+ENV_FILE_PATH = BASE_DIR / ".env"
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "car_owners.settings")
+dotenv.read_dotenv(str(ENV_FILE_PATH))
```

Comment:
I would recommend to handle environment in environment setup, not in code. You can source .env in Dockerfile or set it in docker-compose yaml